dataverse-webapp/src/test/resources/exportdata/testDatacite.xml

```diff
@@ -1,18 +1 @@
-<resource xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4/metadata.xsd"
-          xmlns="http://datacite.org/schema/kernel-4"
-          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-    <identifier identifierType="DOI">10.5072/FK2/05NAR1</identifier>
-    <creators><creator><creatorName>Admin, Dataverse</creatorName><affiliation affiliationIdentifier="https://ror.org/024yc3q36" affiliationIdentifierScheme="ROR">Dataverse.org</affiliation></creator></creators>
-    <titles>
-        <title>Export test</title>
-    </titles>
-    <publisher>Root</publisher>
-    <publicationYear>2019</publicationYear>
-    <resourceType resourceTypeGeneral="Dataset"/>
-    
-    <descriptions>
-        <description descriptionType="Abstract">EXPORT</description>
-    </descriptions>
-    <contributors><contributor contributorType="ContactPerson"><contributorName>Admin, Dataverse</contributorName><affiliation>Dataverse.org</affiliation></contributor></contributors>
-    <fundingReferences><fundingReference><funderName>Grant Agency</funderName><funderIdentifier funderIdentifierType="ROR">https://ror.org/00e0c0q64</funderIdentifier><awardNumber>grant-123</awardNumber></fundingReference></fundingReferences>
-</resource>
+<resource xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4/metadata.xsd" xmlns="http://datacite.org/schema/kernel-4" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><identifier identifierType="DOI">10.5072/FK2/05NAR1</identifier><creators><creator><creatorName>Admin, Dataverse</creatorName><affiliation affiliationIdentifier="https://ror.org/024yc3q36" affiliationIdentifierScheme="ROR">Dataverse.org</affiliation></creator></creators><titles><title>Export test</title></titles><publisher>Root</publisher><publicationYear>2019</publicationYear><resourceType resourceTypeGeneral="Dataset"/><descriptions><description descriptionType="Abstract">EXPORT</description></descriptions><contributors><contributor contributorType="ContactPerson"><contributorName>Admin, Dataverse</contributorName><affiliation>Dataverse.org</affiliation></contributor></contributors><fundingReferences><fundingReference><funderName>Grant Agency</funderName><funderIdentifier funderIdentifierType="ROR">https://ror.org/00e0c0q64</funderIdentifier><awardNumber>grant-123</awardNumber></fundingReference></fundingReferences></resource>
```

Comment:
Actually: `assertThat()..isEqualToIgnoringWhitespace` have one weak point. It will not check whitespaces in tag values (`<tag> test test</tag>` is not the same as `<tag>testtest</tag>`. But we can leave it as is right now. Although in future we should talk about how we want to handle xml comparing (maybe we could find some suitable library)